app/src/main/res/layout/activity_group.xml

```diff
@@ -77,29 +77,36 @@
                 android:layout_width="@dimen/menu_avatar_size"
                 android:layout_height="@dimen/menu_avatar_size"
                 android:background="@drawable/circle_fill_grey"
-                android:src="@drawable/ic_face_white_48dp" />
+                android:src="@drawable/ic_face_white_48dp"/>
 
             <de.hdodenhof.circleimageview.CircleImageView
                 style="@style/MenuAvatarStyle"
                 android:layout_width="@dimen/menu_avatar_size"
                 android:layout_height="@dimen/menu_avatar_size"
                 android:background="@drawable/circle_fill_grey"
-                android:src="@drawable/ic_face_white_48dp" />
+                android:src="@drawable/ic_face_white_48dp"/>
 
             <de.hdodenhof.circleimageview.CircleImageView
                 style="@style/MenuAvatarStyle"
                 android:layout_width="@dimen/menu_avatar_size"
                 android:layout_height="@dimen/menu_avatar_size"
                 android:background="@drawable/circle_fill_grey"
-                android:src="@drawable/ic_face_white_48dp" />
+                android:src="@drawable/ic_face_white_48dp"/>
 
-            <de.hdodenhof.circleimageview.CircleImageView
+            <com.compscieddy.meetinthemiddle.FontTextView
                 android:id="@+id/invite_button"
-                style="@style/MenuAvatarStyle"
-                android:layout_width="@dimen/menu_avatar_size"
-                android:layout_height="@dimen/menu_avatar_size"
-                android:background="@drawable/circle_fill_grey"
-                android:src="@drawable/ic_plus_white_48dp" />
+                android:layout_width="wrap_content"
```

Comment:
Let's actually move some of these properties out into a @style - check @styles/MenuAvatarStyle for an example. Try to move out common properties that make sense for something like this "ButtonOutlineAccent"